src/ios/MusicControls.m

```diff
@@ -252,17 +277,22 @@ - (void) deregisterMusicControlsEventListener {
     [[NSNotificationCenter defaultCenter] removeObserver:self name:@"receivedEvent" object:nil];
     [self setLatestEventCallbackId:nil];
 
-    if (floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_9_0) {
```

Comment:
Same question as above

Reply:
A testing mistake. I will add that back.